app/src/main/java/net/skyscanner/backpack/demo/compose/SingleSelectChipGroupStory.kt

```diff
@@ -0,0 +1,142 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import net.skyscanner.backpack.compose.chip.BpkChipStyle
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.singleselectchipgroup.BpkSingleSelectChipGroup
+import net.skyscanner.backpack.compose.singleselectchipgroup.SingleSelectChipGroupData.ChipGroupType
+import net.skyscanner.backpack.compose.singleselectchipgroup.SingleSelectChipGroupData.ChipItem
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.compose.tokens.Deals
+import net.skyscanner.backpack.compose.tokens.Heart
+import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.demo.components.SingleSelectChipGroupComponent
+import net.skyscanner.backpack.demo.meta.ComposeStory
+
+@Composable
+@SingleSelectChipGroupComponent
+@ComposeStory("Default")
+fun SingleSelectChipGroupStoryDefault(modifier: Modifier = Modifier) =
+    SingleSelectChipGroupDemo(BpkChipStyle.Default, modifier)
+
+@Composable
+@SingleSelectChipGroupComponent
+@ComposeStory("On Dark")
+fun SingleSelectChipGroupStoryOnDark(modifier: Modifier = Modifier) =
+    SingleSelectChipGroupDemo(BpkChipStyle.OnDark, modifier)
+
+@Composable
+@SingleSelectChipGroupComponent
+@ComposeStory("On Image")
+fun SingleSelectChipGroupStoryOnImage(modifier: Modifier = Modifier) =
+    SingleSelectChipGroupDemo(BpkChipStyle.OnImage, modifier)
+
+@Composable
+@SingleSelectChipGroupComponent
+internal fun SingleSelectChipGroupDemo(style: BpkChipStyle, modifier: Modifier = Modifier) {
+
+    Box(modifier) {
+        if (style == BpkChipStyle.OnImage) {
+            Image(
+                painter = painterResource(R.drawable.city),
+                contentDescription = null,
+                contentScale = ContentScale.Crop,
+                modifier = Modifier.fillMaxSize(),
+            )
+        }
+        val background = when (style) {
+            BpkChipStyle.Default -> Color.Transparent
+            BpkChipStyle.OnDark -> BpkTheme.colors.surfaceContrast
+            BpkChipStyle.OnImage -> Color.Transparent
+        }
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(background)
+                .padding(BpkSpacing.Base),
+            verticalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
+        ) {
+            SingleSelectChipGroupSample(type = ChipGroupType.RAIL, style = style)
+            Spacer(modifier = Modifier.height(BpkSpacing.Xxl))
+            SingleSelectChipGroupSample(type = ChipGroupType.WRAP, style = style)
+        }
+    }
+}
+
+@Composable
+@SingleSelectChipGroupComponent
+
+internal fun SingleSelectChipGroupSample(
+    type: ChipGroupType,
+    modifier: Modifier = Modifier,
+    selectedIndex: Int = -1,
```

Comment:
something i didn't spot before - we shouldn't allow a non-selected item aka -1/out of bounds - you always need a selected item for the single select group as otherwise it's not a valid state.